SPECS/onefuzz/onefuzz.spec

```
@@ -0,0 +1,55 @@
+Summary:        A self-hosted Fuzzing-As-A-Service platform
+Name:           onefuzz
+Version:        8.5.0
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Tools
+URL:            https://github.com/microsoft/OneFuzz
+Source0:        https://github.com/microsoft/onefuzz/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
+Patch0:         git-version.patch
+
+BuildRequires:  cargo >= 1.68
+BuildRequires:  git
+BuildRequires:  kernel-headers
+BuildRequires:  liblzma-devel
+BuildRequires:  libunwind-devel
+BuildRequires:  openssl-devel
+BuildRequires:  perl
+BuildRequires:  rust >= 1.68
+BuildRequires:  util-linux
+
+%description
+Project OneFuzz enables continuous developer-driven fuzzing to proactively
+harden software prior to release. With a single command, which can be baked
+into CICD, developers can launch fuzz jobs from a few virtual machines to
+thousands of cores.
+
+%prep
+%setup -q
+%patch0 -p1
```

I think it would be cleaner to leverage autosetup:
```suggestion
%autosetup -p1
```